vita3k/gui/src/content_manager.cpp

```diff
@@ -72,12 +72,12 @@ size_t get_app_size(GuiState &gui, EmuEnvState &emuenv, const std::string &app_p
 
 std::string get_unit_size(const size_t &size) {
     std::string size_str;
-    if (size >= GB(10))
-        size_str = std::to_string(size / GB(1)) + " GB";
-    else if (size >= MB(10))
-        size_str = std::to_string(size / MB(1)) + " MB";
-    else if (size >= KB(1))
-        size_str = std::to_string(size / KB(1)) + " KB";
+    if (size >= GiB(10))
+        size_str = std::to_string(size / GiB(1)) + " GiB";
+    else if (size >= MiB(10))
+        size_str = std::to_string(size / MiB(1)) + " MiB";
+    else if (size >= KiB(1))
+        size_str = std::to_string(size / KiB(1)) + " KiB";
```

Comment:
revert, i using just same text of psvita, vita no write i

Reply:
I wany keep string if vita

Reply:
the vita says "(size in MiB) MB", which is wrong you don't want to tell the user the game is using "1000 MB" when in reality its using "1000 MiB", it's like having a math test and the question is 2+2, you know the answer is 4, but because your friend puts 5 you put 5 instead of 4. :brain:

Reply:
no really care, just using MB like psvita, i no speek other things, just keep official text, that ,all

Reply:
i won :sunglasses:

Reply:
🤷‍♂️